sdk/appconfiguration/tests.yml

```diff
@@ -15,15 +15,3 @@ jobs:
         AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
         AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
         AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
-      PostRunSteps:
-        - script: |
-            python --version
-            python -m pip install setuptools termcolor in_place
-            python sdk/spring/scripts/replace_util.py --module appconfiguration --log debug --color false
-          displayName: 'Run replace_util.py'
-        - task: Maven@3
-          displayName: 'Run tests again.'
```

Comment:
Why did we have this step before?

Reply:
Previously, both Spring 2.2 and 2.3 were being supported by the same code base, so in testing, because Maven can only support a single dependency in the POM, the dependency version would be toggled after the first test run passed with 2.3 to version 2.2 to validate that as well. Moving forward this will no longer be the case as underlying dependencies between 2.4 and 2.3 have major breaking changes and this infrastructure will no longer be used.